### PR TITLE
Changes to ovn daemonset

### PR DESCRIPTION
**Pull request by pecameron**
Respond to comments in PR-351

With these changes an image can be built, the cluster provisioned for ovn and ovs/ovn brought up and run.

Signed-off-by: pcameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@rajatchopra @dcbw @shettyg The PR permtits an image to be built and included in daemonsets that run ovs, ovnkube-master, and ovnkube PTAL

File: dist/images/Dockerfile [hunk 1]
```diff
@@ -1,44 +1,54 @@
...
+	libnuma.so \
+	openssl \
+	numactl-libs \
+	firewalld-filesystem \
+	libpcap \
+	iproute strace socat && \
+	yum clean all
+
+# get a reasonable version of openvswitch
+RUN rpm -i http://cbs.centos.org/kojifiles/packages/openvswitch/2.9.0/4.el7/x86_64/openvswitch-2.9.0-4.el7.x86_64.rpm
```
Comment (adammhaile) [inline dist/images/Dockerfile]: @pecameron For images built for OCP, you *cannot* pull RPMs from external sources. They *will* fail QE security checks. All RPMs must come from internal, vetted sources.

**Comment by adammhaile** [comment]
Any ETA on this being fixed and merged?

File: dist/images/Dockerfile (hunk 1 above)
Comment (pecameron) [reply dist/images/Dockerfile]: @adammhaile This set of changes builds the community image, not the OCP image. I thought the OCP image was going to be built in the dist-git repo that was recently created. I will need some help in getting a proper Dockerfile for the OCP image.

**Comment by pecameron** [comment]
@rajatchopra @dcbw @shettyg PTAL - I am back from PTO and this is blocking me creating the OCP image.
@adammhaile I am waiting for reviews and a merge.

File: dist/images/Dockerfile (hunk 1 above)
Comment (adammhaile) [reply dist/images/Dockerfile]: @pecameron Ah, right... that will be from http://github.com/openshift/ose-ovn-kubernetes
Sorry for confusion. We will pull the dockerfile and related files from that repo and put them into the dist-git then build from there. Let me know when you need help with the OCP version of the dockerfile.

**Comment by pecameron** [comment]
@rajatchopra  Hi Rajat,  @shettyg noticed the author is root. I have a change, but you have already merged. Is there any way this can be fixed?

**Comment by shettyg** [comment]
@pecameron I fixed it backend. For the next commits, have the same Author name as your Signed-off-by name. Thanks

**Comment by pecameron** [comment]
@shettyg Thanks for fixing this. I was working on a different than machine than usual and didn't notice the author. I will pay attention to this in the future. Again, thanks.

**Pull request merged**